beta-prompts/beta_prompts/templating_framework.py

```
@@ -0,0 +1,1029 @@
+
+        self.jinja_env.filters["format_list"] = format_list
+        self.jinja_env.filters["xml_wrap"] = xml_wrap
+        self.jinja_env.filters["conditional_section"] = conditional_section
+
+    def _initialize_builtin_templates(self):
+        """Initialize built-in template collection"""
+
+        # Comprehensive Analysis Template
+        analysis_template = """
+{%- set role_definition -%}
+You are an expert {{ domain }} analyst with {{ experience|default("10") }} years of experience in {{ specialization|default("comprehensive analysis") }}.
```

Ruff (0.12.2)

Line too long (153 > 100)

(E501)